.github/workflows/pull_request.yml

```diff
@@ -79,10 +80,18 @@ jobs:
           fetch-depth: 0
 
       - name: build images
-        run: make build-images
+        run: |
+         IMAGE_TAG="int-test"
+         BPFMAN_AGENT_IMG="${{ env.WF_BPFMAN_AGENT_IMG }}"
+         BPFMAN_OPERATOR_IMG="${{ env.WF_BPFMAN_OPERATOR_IMG}}"
+         MULTIARCH_TARGETS="${{ env.WF_MULTIARCH_TARGETS }}" make build-images
```

Comment:
Do we need to build all arch images for the integration tests if they're only running on x86?

Reply:
Oh NM I see above that amd64 is the only one